module-api/src/main/java/com/cosmetics/api/goods/controller/GoodsController.java

```diff
@@ -19,17 +20,32 @@ public class GoodsController {
 
     @GetMapping(value = "/{goodsNo}")
     public GoodsManagementResponse findGoods(@PathVariable Long goodsNo) {
-        return goodsService.findByGoodsNo(goodsNo);
+        GoodsManagement goodsManagement = goodsService.findByGoodsNo(goodsNo);
+        //dto -> responseDto
+        GoodsManagementResponse responseGoodsManagement = GoodsManagementResponse.toResponseDto(goodsManagement);
```

Comment:
```suggestion
        GoodsManagementResponse responseGoodsManagement = GoodsManagementResponse.fromEntity(goodsManagement);
```
??

https://github.com/f-lab-edu/pat-pat-cosmetics/blob/1d85832c3a0ecc20f646fc00ace69fb6557e7f2e/module-domain/src/main/java/com/cosmetics/domain/goods/dto/GoodsManagementResponse.java#L66

Reply:
@f-lab-lyan 
저번 리뷰 때 서비스에서 response용 dto를 컨트롤러로 빼기로해서 옮겼는데요 fromEntity로 네이밍을 하면되는건가요?